ftplugin/go/commands.vim

```diff
@@ -9,6 +9,8 @@ nnoremap <silent> <Plug>(go-run) :<C-u>call go#cmd#Run(expand('%'))<CR>
 nnoremap <silent> <Plug>(go-build) :<C-u>call go#cmd#Build('')<CR>
 nnoremap <silent> <Plug>(go-install) :<C-u>call go#cmd#Install()<CR>
 nnoremap <silent> <Plug>(go-test) :<C-u>call go#cmd#Test(0, '')<CR>
+nnoremap <silent> <Plug>(go-test-focused) :<C-u>call go#cmd#TestFocused('')<CR>
```

Comment:
Let us rename all `Focused` to `Function`. So the internal functions should be renamed too. The command should be `:GoTestFunction`. It's more simpler and obvious what it does.